oshdb-util/src/main/java/org/heigit/ohsome/oshdb/util/geometry/Geo.java

```diff
@@ -164,7 +164,7 @@ public static double areaOf(Polygon poly) {
     for (int i = 0; i < poly.getNumInteriorRing(); i++) {
       area -= Math.abs(ringArea((LinearRing) poly.getInteriorRingN(i)));
     }
-    return area;
+    return Math.max(0, area);
```

Comment:
You could specify here or in the java doc of the method why you use the Math.max function or that the area cannot be < 0 . It is just a thought, you don't have to.

Reply:
Good thought. Ideally this would be reflected in the [ohsome API docs](https://github.com/GIScience/ohsome-api/tree/master/docs) as well.

Reply:
I've added a sentence about it to the javadoc. not sure if it really needs to be mentioned anywhere else, since I would thing that everybody would assume that areas can never be negative values.